### PR TITLE
feat: notebooks wip

### PR DESCRIPTION
**Pull request by drdelambre**
current status:
Kapture 2020-05-06 at 16 07 03

### CODE REVIEW

File: ui/src/pageLayout/constants/navigationHierarchy.ts [hunk 1]
```diff
@@ -121,6 +121,19 @@ export const generateNavItems = (orgID: string): NavItem[] => {
       },
       activeKeywords: ['data-explorer'],
     },
+    {
+      id: 'notebooks',
+      testID: 'nav-item-notebooks',
+      icon: IconFont.Erlenmeyer,
```
Comment (alexpaxton) [inline ui/src/pageLayout/constants/navigationHierarchy.ts:127]: Dat flask tho

File: ui/src/notebooks/components/Notebook.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import {NotebookProvider, NotebookContext} from 'src/notebooks/notebook.context'
+
+const NotebookHeader: FC = () => {
+  //const { id } = useContext(NotebookContext)
+  //const { context, updateContext } = useContext(TimeContext)
+
+  return (
+    <>
+      <h1>NOTEBOOKS</h1>
+    </>
```
Comment (alexpaxton) [inline ui/src/notebooks/components/Notebook.tsx:15]: I think you can just do `<Page.Title title="Notebooks" />`

File: ui/src/pageLayout/constants/navigationHierarchy.ts (hunk 1 above)
Comment (drdelambre) [reply ui/src/pageLayout/constants/navigationHierarchy.ts:127]: ...so good...

**Comment by drdelambre** [comment]
wanted to see how we can reuse resources within the redux state
Kapture 2020-05-07 at 15 32 46

**Comment by drdelambre** [comment]
rough query on query action. feels like we have to tell the user what cell to reference and introduce them to the concept of linking somehow

Kapture 2020-05-07 at 16 55 53

**Comment by drdelambre** [comment]
making an interface that's a little lighter weight, now you can extend queries across visualizations with a global `__PREVIOUS_RESULT__` flux variable. this gets reduced into a collection of static queries that can be run in parallel and then have their results attached back to their original visualization / stage.


Kapture 2020-05-12 at 14 21 53

**Pull request closed without merging**